internal/boxcli/secrets.go

```diff
@@ -0,0 +1,211 @@
+// Copyright 2023 Jetpack Technologies Inc and contributors. All rights reserved.
+// Use of this source code is governed by the license in the LICENSE file.
+
+package boxcli
+
+import (
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"go.jetpack.io/devbox/internal/devbox"
+	"go.jetpack.io/devbox/internal/devbox/devopt"
+	"go.jetpack.io/envsec/pkg/envsec"
+)
+
+type secretsFlags struct {
+	config configFlags
+}
+
+type secretsInitCmdFlags struct {
+	force bool
+}
+
+type secretsListFlags struct {
+	show   bool
+	format string
+}
+
+func secretsCmd() *cobra.Command {
+	flags := &secretsFlags{}
+	cmd := &cobra.Command{
+		Use:     "secrets",
+		Aliases: []string{"envsec"},
+		Short:   "Interact with devbox secrets.",
+	}
+	cmd.AddCommand(secretsInitCmd(flags))
+	cmd.AddCommand(secretsListCmd(flags))
+	cmd.AddCommand(secretsRemoveCmd(flags))
+	cmd.AddCommand(secretsSetCmd(flags))
+	cmd.Hidden = true
+
+	flags.config.registerPersistent(cmd)
```

Comment:
need to update the `envsec` wording to `secrets` in:
https://github.com/jetpack-io/devbox/blob/11bb6f05a4562d16ee572e54a11c13a6c9943346/internal/boxcli/config.go#L21-L30